interface/src/app/gene/network/network.less

```diff
@@ -54,3 +54,11 @@ text.legend-title {
 .rzslider {
   width: 600px;  // Default: 100%
 }
+
+#gene-slider .rz-pointer {
+  background-color: magenta;
+}
+
+#gene-slider .rz-selection {
+  background: magenta;
+}
```

Comment:
I would not have chosen `magenta` here. Maybe we can have Deepa figure out a standardized color palate for us to work with?

Reply:
I played with a few colors and eventually selected "magenta" because I want to use a color that is:
(1) different from "blue" (which is used by the first slider);
(2) visually obvious;
(3) not included in the legend bar.

I will talk to Deepa.

Reply:
This could be an issue for follow-up. Others may disagree with me about the use of magenta.